### PR TITLE
Younesbessa/instagram dark welcome page

### PR DESCRIPTION
**Pull request by YounesBessa**

### CODE REVIEW

File: src/screens/Projects/InstagramDark/Welcome/index.tsx [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+import Link from "@components/Link";
+
+const Welcome: NextPage = () => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Background />
+      <MainContainer>
+        <Head
```
Comment (AntoineKM) [inline src/screens/Projects/InstagramDark/Welcome/index.tsx:17]: Update Head component with the ef one

Comment (AntoineKM) [reply src/screens/Projects/InstagramDark/Welcome/index.tsx:17]: and use no index no follow we dont want this page to be on google

File: src/screens/Projects/InstagramDark/Welcome/index.tsx [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+const SocialList = styled.div`
+  display: flex;
+  margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+  width: calc(100% - 15px * 2);
+  margin-top: 10px;
+`;
+
+const SocialLink = styled(Link)`
```
Comment (AntoineKM) [inline src/screens/Projects/InstagramDark/Welcome/index.tsx:201]: You can use

```
const SocialButton = styled(({ children, ...props }) => (
  <Button as={Link} {...props}>
    {children}
  </Button>
))<{ backgroundColor: string }>`
   ......css.....
   ${({ backgroundColor}) => backgroundColor && `background-color: ${backgroundColor}`}
`;
```

So you can have a reduced and proper code like:

```
<SocialButton
  href={"https://example.com"}
  backgroundColor={"#123456"}
>Example</SocialButton>
```

**Review by AntoineKM — CHANGES_REQUESTED** [review]

**Review by AntoineKM — CHANGES_REQUESTED** [review]
Il faut également que tu ajoutes le disclaimer

"This extension automatically add our company account to your following list, you can simply unfollow us but it's give us support because this is entirely free app, we do not steal your personnal data, check the open source code [here](https://outgoing.prod.mozaws.net/v1/06f268f7f7613f4a2778b52677190215385b350d0d1902c3438c6aaa276e5727/https%3A//github.com/onRuntime/instagram-dark-extension)."

**Comment by YounesBessa** [comment]
> Also add the disclaimer
> 
> "This extension automatically add our company account to your following list, you can simply unfollow us but it's give us support because this is entirely free app, we do not steal your personnal data, check the open source code [here](https://outgoing.prod.mozaws.net/v1/06f268f7f7613f4a2778b52677190215385b350d0d1902c3438c6aaa276e5727/https%3A//github.com/onRuntime/instagram-dark-extension)."

Done

**Review by AntoineKM — APPROVED** [review]

**Pull request merged**